tests/testthat/test-modify-model-field.R

```diff
@@ -71,6 +71,10 @@ test_that("replace_tag() works correctly", {
   # test_replacing
   new_mod <- replace_tag(new_mod, ORIG_TAGS[1], NEW_TAGS[1])
   expect_identical(new_mod[[YAML_TAGS]], c(NEW_TAGS[1], ORIG_TAGS[2:length(ORIG_TAGS)]))
+
+  # check that YAML is modified
```

Comment:
Strictly speaking, this is a test of `replace_tag()`, not `replace_model_field()`. Because `replace_model_field()` is exported, it may be better to test it directly.